package.json

```diff
@@ -15,9 +15,9 @@
     "start:ci": "cross-env POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub TTA_SMART_HUB_URI=http://localhost:3000 concurrently \"yarn start:web\" \"yarn client\" \"yarn start:worker\"",
     "start:web": "node ./build/server/index.js",
     "start:worker": "node ./build/server/worker.js",
-    "server": "nodemon src/index.js --exec babel-node --ignore ./tests/ --ignore ./playwright-report/ --ignore ./test-results/",
-    "worker": "nodemon src/worker.js --exec babel-node --ignore ./tests/ --ignore ./playwright-report/ --ignore ./test-results/",
-    "server:debug": "nodemon --inspect=0.0.0.0:9229 src/index.js --exec babel-node --ignore ./tests/ --ignore ./playwright-report/ --ignore ./test-results/",
+    "server": "nodemon src/index.js --exec babel-node --ignore ./tests/ --ignore ./playwright-report/ --ignore ./test-results/ --ignore ./frontend/",
+    "worker": "nodemon src/worker.js --exec babel-node --ignore ./tests/ --ignore ./playwright-report/ --ignore ./test-results/ --ignore ./frontend/",
```

Comment:
nodemon was reloading every time the frontend files changed

Reply:
Yea was wondering the cause for this.